File: Programmers/Lv0/한번만등장한문자.py
def solution(s):
    answer = ''
    dict = {}
    for ch in s:
        if ch in dict:
            dict[ch] += 1
        else :
            dict[ch] = 1
    for key in dict:
        if dict[key] == 1:
            answer += key
    return ''.join(sorted(answer))

File: Programmers/Lv0/한번만등장한문자.cpp
#include <string>
#include <vector>
#include <map>
using namespace std;

string solution(string s) {
    string answer = "";
    map <char, int> _map;
    for (char ch : s)
        _map[ch] += 1;
    for (auto data : _map)
        if (data.second == 1)
            answer += data.first;
    return answer;
}